amenity-points.mss

```diff
@@ -2051,14 +2136,102 @@
+  [feature = 'office_it'],
+  [feature = 'office_lawyer'],
+  [feature = 'office_logistics'],
+  [feature = 'office_moving_company'],
+  [feature = 'office_notary'],
+  [feature = 'office_physician'],
+  [feature = 'office_private_investigator'],
+  [feature = 'office_property_management'],
+  [feature = 'office_surveyor'],
+  [feature = 'office_tax_advisor'],
+  [feature = 'office_therapist'],
+  [feature = 'office_travel_agent'] {
+    [zoom >= 18],
+    [way_pixels > 3000][zoom >= 17] {
+      text-name: "[name]";
+      text-size: @standard-font-size;
+      text-wrap-width: @standard-wrap-width;
+      text-line-spacing: @standard-line-spacing-size;
+      text-dy: 8;
+      text-fill: @shop-text;
+      text-face-name: @standard-font;
+      text-halo-radius: @standard-halo-radius;
+      text-halo-fill: rgba(255, 255, 255, 0.6);
+      text-placement: interior;
+    }
+  }
+
+  // government and public institutions
+  [feature = 'office_administrative'],
+  [feature = 'office_adoption_agency'],
+  [feature = 'office_educational_institution'],
+  [feature = 'office_employment_agency'],
+  [feature = 'office_government'],
+  [feature = 'office_ngo'],
+  [feature = 'office_political_party'],
+  [feature = 'office_quango'],
+  [feature = 'office_religion'],
+  [feature = 'office_tax'] {
+    [zoom >= 18],
+    [zoom >= 17][way_pixels > 3000] {
+      text-name: "[name]";
+      text-size: @standard-font-size;
+      text-wrap-width: @standard-wrap-width;
+      text-line-spacing: @standard-line-spacing-size;
+      text-dy: 8;
+      text-fill: @public-service;
+      text-face-name: @standard-font;
+      text-halo-radius: @standard-halo-radius;
+      text-halo-fill: rgba(255, 255, 255, 0.6);
+      text-placement: interior;
+    }
+  }
+
+  // offices
+  [feature = 'office_association'],
+  [feature = 'office_charity'],
+  [feature = 'office_company'],
+  [feature = 'office_energy_supplier'],
+  [feature = 'office_financial'],
+  [feature = 'office_forestry'],
+  [feature = 'office_foundation'],
+  [feature = 'office_newspaper'],
+  [feature = 'office_research'],
+  [feature = 'office_telecommunication'],
+  [feature = 'office_water_utility'],
+  [feature = 'office_yes'] {
+    [zoom >= 18],
+    [zoom >= 17][way_pixels > 3000] {
+      text-name: "[name]";
+      text-size: @standard-font-size;
+      text-wrap-width: @standard-wrap-width;
+      text-line-spacing: @standard-line-spacing-size;
+      text-dy: 8;
+      text-fill: @office;
+      text-face-name: @standard-font;
+      text-halo-radius: @standard-halo-radius;
+      text-halo-fill: rgba(255, 255, 255, 0.6);
+      text-placement: interior;
+    }
+  }
+
   [feature = 'shop_supermarket'],
   [feature = 'shop_department_store'] {
     [zoom >= 16] {
       text-name: "[name]";
       text-size: @standard-font-size;
       text-wrap-width: @standard-wrap-width;
       text-line-spacing: @standard-line-spacing-size;
-      text-dy: 12;
+      text-dy: 8;
```

Comment:
This change is outside the scope of the `office=*` rendering and makes the change with shops, which can disable rendering some labels, especially with high glyphs (like "É"), so should be probably reverted.